### PR TITLE
Initial layout of repository

### PR DESCRIPTION
**Pull request by halmamol**
This PR introduces the initial layout of the software documentation repository. It adds a docs folder with all the source files for the documentation. Some text is written in them, just as guideline, official documentation will be included in further PRs. 

The shape of the repository in read-the-docs would look like this: https://test-sw-docs.readthedocs.io/en/initial_layout/NEXUS.html (this link will be erased once PR is approved)

### CODE REVIEW

File: docs/source/IC.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+IC
+=====
+
+IC stands for Invisible Cities
+*Language*: Python; *Based*: Geant4 toolkit, divided in logical blocks
```
Comment (paolafer) [inline docs/source/IC.rst]: It's a repetition of the explanation of NEXUS.

Comment (halmamol) [reply docs/source/IC.rst]: Fixed!

File: docs/source/contact.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Are you missing something? Would you like to contribute in this documentation?
+Contact any of us:
+
+.. list-table:: Title
+   :widths: 25 75
+   :header-rows: 1
+
+   * - Contact Person
+     - Organisation Task
+   * - `Jose Angel Hernando <jose.hernando@usc.es>`_
```
Comment (paolafer) [inline docs/source/contact.rst:13]: I can't see the names, can you?

Comment (halmamol) [reply docs/source/contact.rst:13]: Yes, sorry, I was trying to do add a table but it seems it didn't work. I don't want to add too many commits on this PR while fixing it. If you want I can totally erase it and add it in a future PR with the complete content of the wiki.

Comment (paolafer) [reply docs/source/contact.rst:13]: No, it's fine, I was just wondering why I couldn't visualize the emails although they are written in the `.rst`. I can approve the PR as it is, anyway.

**Review by paolafer — APPROVED** [review]
First layout approved!

**Pull request merged**